src/reducers/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from './filterReducer';
```

Comment:
resource...

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a